judge/chroot-settings.sh.in

```diff
@@ -0,0 +1,7 @@
+#/bin/sh
+
+# Chroot subdirs needed: (optional lib64 only needed for amd64 architecture)
+SUBDIRMOUNTS="etc usr lib bin"
+if [ "$(uname -m)" = "x86_64" ]; then
+	SUBDIRMOUNTS="$SUBDIRMOUNTS lib64"
+fi
```

Comment:
I'd rather not move these settings to a separate file, just to be able to reference them in the `dj_run_chroot` script. Instead just print a generic warning there that not all paths are included in the chroot used for judging, and that the actual list can be found in the `chroot-start-stop.sh` script.

Reply:
I've added this to the text. I still prefer to have a list in the prompt and people can check the script if they suspect it got outdated.